cfgov/jinja2/v1/ask-cfpb/answer-page.html

```diff
@@ -112,19 +112,19 @@ <h3 class="answer-snippet">
 
 {% block content_sidebar scoped  %}
 
-    {% if page.answer_base.next_step %}
+    {% if page.related_resource %}
         <div class="block block__flush-top">
             <h3 >
                 Explore related resources
             </h3>
-            {{ page.answer_base.next_step.text | safe }}
+            {{ page.related_resource.text | safe }}
         </div>
     {% endif %}
     {% if page.sidebar_panels %}
             {{ streamfield_sidefoot.render(page.sidebar) }}
     {% endif %}
     {% if category and subcategories %}
-        <div class="block {% if not page.answer_base.next_step %}block__flush-top{% endif %}">
+        <div class="block {% if not page.next_step %}block__flush-top{% endif %}">
```

Comment:
I think this `next_step` reference also needs to be updated to `page.related_resource`.

Reply:
great catch.

Reply:
Should we test this type of thing?  Like - if there exists a `related_resource` we expect to see it on the rendered page?  Not sure if that is overkill but we do something similar with our molecules tests.